micromasters/views.py

```diff
@@ -0,0 +1,15 @@
+"""
+micromasters views
+"""
+from django.contrib.auth.views import redirect_to_login
+from django.urls import reverse
+
+
+def cms_signin_redirect_to_site_signin(request):
+    """Redirect wagtail admin signin to site signin page"""
+    return redirect_to_login(reverse('wagtailadmin_home'), login_url="/")
+
+
+def cms_password_reset_redirect_to_site_signin(request):
+    """Redirect wagtail admin password reset to site signin page"""
+    return redirect_to_login(reverse('wagtailadmin_home'), login_url="/")
```

Comment:
These should live in the `cms` app. It does not make sense to create a new `micromasters` app just for these redirects. And given that these redirects are for the `cms` and we already have a `cms` app it makes even less sense to create the `micromasters` app for this.